config/crd/bases/management.project-flotta.io_edgedevices.yaml

```diff
@@ -118,6 +118,20 @@ spec:
                         type: integer
                     type: object
                 type: object
+              metricsReceiver:
+                properties:
+                  ? "R\requestNumSamples"
+                  : description: request num samples
+                    format: int64
+                    type: integer
+                  timeoutSeconds:
+                    description: timeout seconds
+                    format: int64
+                    type: integer
+                  url:
+                    description: url
```

Comment:
If you add description, please describe what the property means/what it is for.

Reply:
It was added automatically by generate

Reply:
It takes it from source code comment.

Reply:
fixed